src/NuGet.Clients/NuGet.VisualStudio.Common/ErrorListTableDataSource.cs

```diff
@@ -10,19 +10,18 @@
 using Microsoft.VisualStudio.Shell.Interop;
 using Microsoft.VisualStudio.Shell.TableControl;
 using Microsoft.VisualStudio.Shell.TableManager;
-using NuGet.VisualStudio;
-using IAsyncServiceProvider = Microsoft.VisualStudio.Shell.IAsyncServiceProvider;
 
```

Comment:
Do you need this extra line here?

Reply:
Not really. I like it this way :)

---
In reply to: [462625126](https://github.com/NuGet/NuGet.Client/pull/3544#discussion_r462625126) [](ancestors = 462625126)